lib/superstore/base.rb

```diff
@@ -32,6 +32,7 @@ class Base
     include ActiveRecord::AutosaveAssociation
     include ActiveRecord::Reflection
     include ActiveRecord::Transactions
+    extend ActiveRecord::Translation
```

Comment:
I am trying to make this look exactly the same as ActiveRecord::Base. This belongs below `extend Querying`.